server/issue.go

```diff
@@ -1021,6 +1027,9 @@ func getDatabaseNameAndStatement(dbType db.Type, databaseName, characterSet, col
 		} else {
 			stmt = fmt.Sprintf("CREATE DATABASE \"%s\" ENCODING %q LC_COLLATE %q;", databaseName, characterSet, collation)
 		}
+		// Set the database owner.
+		// For tenant project, the schema for the newly created database will belong to the same owner.
+		stmt = fmt.Sprintf("%s\nALTER DATABASE \"%s\" OWNER TO %s;\n", stmt, databaseName, owner)
```

Comment:
we can directly use WITH OWNER in CREATE DATABASE to avoid multiple statements

Reply:
That's my first attempt but WITH OWNER requires a membership check.

Reply:
That's quite bizarre. What's the logic behind CREATE WITH OWNER requires membership check, while ALTER xxx OWNER TO doesn't?

The logic thinking is they both require the same membership check, as one can not arbitrarily transfer the ownership to another user unless they have membership relationship (or the operator is a superuser)

Reply:
Added a comment to explain why we use ALTER DATABASE OWNER TO.